frontend/packages/erd-core/src/components/ERDRenderer/ERDRenderer.module.css

```diff
@@ -18,7 +18,7 @@
 
 .main {
   width: 100%;
-  height: 100%;
+  height: calc(100svh - var(--default-header-height));
```

Comment:
I tried using ``vh`` instead of ``svh``, but it didn't display correctly, so I went back to using ``svh``.